### DISCUSSION TITLE
pfSense 2.6.0 Beta Syslog processing question

### DISCUSSION THREAD

**Discussion by RagnarSTS** [Security-Onion-Solutions/securityonion#6823] [Unsupported Versions] [answered]
Future versions of pfSense syslog may use RFC 5424 instead of RFC 3164.

Older versions of pfSense syslog send things out with RFC 3164 and SO2 recognizes this fully and processes (event.module.keyword indicated) pfsense module for logstash.  
Using the newer RFC 5424 it doesn't get processed by the pfsense module and simply gets processed by standard syslog.  As of pfSense 2.6.0 betas; the newer standard is available and may become standard in the future. 

While I'm sure that the newer setup is more proper, and the processor is required for the old due to sillyness ... it is kind of nice to have the event logs show the obvious differentiation for 'pfsense' instead of being lumped into syslog events with other equipment.  Is it worth the time to try and edit the pfsense module to recognize the new stuff, or should I just set it back to the old style syslog messages?

**Comment by dougburks** [comment, marked as answer]
I would just set it back to the old style since that's what our pfsense parser is designed for.

**Reply by RagnarSTS** [reply]
I dream of a world where equipment specific parsers are not required.  :)
